wgsl/Makefile

```diff
@@ -1,17 +1,15 @@
+.PHONY: all clean
+
 all: index.html grammar/grammar.js
 
+clean:
+	rm -f index.html index.pre.html grammar/grammar.js
 
-GHC=https://github.com/gpuweb/gpuweb/blob
-index.html: index.pre.html Makefile
-	# Append another line to the "This version:" metadata at the top.
-	sed -e "s,gpuweb/wgsl/</a>,gpuweb/wgsl/</a><br><a href=\"$(GHC)/$(shell git rev-parse HEAD)/wgsl/index.bs\">$(GHC)/$(shell git rev-parse HEAD)/wgsl/index.bs</a>," <index.pre.html >index.html
+index.pre.html: index.bs
+	DIE_ON=everything bash ../tools/invoke-bikeshed.sh index.pre.html index.bs
```

Comment:
I don't think this is correct. We originally did a `sed` to replace content in the file and then use that later. We never built index.pre.html as spec itself which this now does.

Reply:
Ah, I see this happens later in a shell script, ignore me.